pyoos/parsers/ioos/one/describe_sensor.py

```diff
@@ -49,7 +56,7 @@ def get_ioos_def(self, ident, elem_type, ont):
 
     def __init__(self, element):
         """ Common things between all describe sensor requests """
-        if isinstance(element, str):
+        if isinstance(element, bytes):
```

Comment:
I think this breaks py27 compatibility. If you find a good solution, please let me know, this wreaks havoc on my life so much.